internal/helm/repository/chart_repository.go

```diff
@@ -520,3 +521,12 @@ func (r *ChartRepository) RemoveCache() error {
 	}
 	return nil
 }
+
+// VerifyChart verifies the chart against a signature.
+// If no signature is provided, a keyless verification is performed.
+// It returns an error on failure.
+func (r *ChartRepository) VerifyChart(_ context.Context, _ *repo.ChartVersion) error {
+	// no-op
+	// this is a no-op because this is not implemented yet.
+	return nil
```

Comment:
If this is not implemented, the method should really return an error because otherwise people might be tempted to call it and upon not receiving an error believe the chart has been properly verified.